### PR TITLE
2635 update dependencies

### PR DESCRIPTION
**Pull request by nekogami**

### CODE REVIEW

File: mandrill-api.gemspec [hunk 1]
```diff
@@ -1,12 +1,12 @@
 Gem::Specification.new do |s|
     s.name = 'mandrill-api'
-    s.version = '1.0.54'
+    s.version = '1.0.55'
```
Comment (dostaglou) [inline mandrill-api.gemspec:3]: Why the version change?

File: mandrill-api.gemspec [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
     s.summary = 'A Ruby API library for the Mandrill email as a service platform.'
-    s.description = s.summary
-    s.authors = ['Mandrill Devs']
-    s.email = 'community@mandrill.com'
+    s.description = s.summary + "\noriginal repository: https://bitbucket.org/mailchimp/mandrill-api-ruby"
+    s.authors = ['Mandrill Devs', 'current verstion: Shippio\'s dev']
```
Comment (dostaglou) [inline mandrill-api.gemspec:6]: Typo on "version" plus I think Shippio doesn't need to be possessive here.

'current version: Shippio Devs'

Comment (KevinBulme) [inline mandrill-api.gemspec:6]:
```suggestion
    s.authors = ['Mandrill Devs', 'current version: Shippio Devs']
```

File: mandrill-api.gemspec (hunk 1 above)
Comment (nekogami) [reply mandrill-api.gemspec:3]: 2 reasons for that:

- it helps updating internals with the Gemfile
- Apache license 2.0 wants you to indicate all changes made, so updating the version is a nice way to write changelog in the new CHANGELOG.md file

File: mandrill-api.gemspec [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
+    s.email = 'it@shippio.com'
```
Comment (KevinBulme) [inline mandrill-api.gemspec:7]: Didn't know this was a thing :p
TIL again

File: mandrill-api.gemspec (hunk 1 above)
Comment (dostaglou) [reply mandrill-api.gemspec:3]: Ok.

File: mandrill-api.gemspec (hunk 2 above)
Comment (KevinBulme) [reply mandrill-api.gemspec:6]: Did a suggestion at about the same time. Closing this one. No offense @dostaglou 🙏

File: mandrill-api.gemspec (hunk 3 above)
Comment (nekogami) [reply mandrill-api.gemspec:7]: I didn't remember it until I saw it in the group management :D

File: mandrill-api.gemspec (hunk 2 above)
Comment (nekogami) [reply mandrill-api.gemspec]: updated

Comment (nekogami) [reply mandrill-api.gemspec]: updated

**Review by KevinBulme — APPROVED** [review]
[LGTM](https://lgtm.lol/i/8)

**Review by dostaglou — APPROVED** [review]

**Pull request merged**